### PR TITLE
PW-7045: Model generation workflow

### PR DESCRIPTION
**Pull request by michaelpaul**
**Description**

Adds a new Github Actions workflow to generate models based on the the latest version of the specs published at [Adyen/adyen-openapi](https://github.com/Adyen/adyen-openapi). Detected changes are proposed via an automatic PR like this https://github.com/Adyen/adyen-java-api-library/pull/810. 

**TODO**

- [x] Update bot access tokens
- [x] Remove `pull_request` event before merging

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,74 @@
+openapi-generator-version:=6.0.1
+openapi-generator-url:=https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/$(openapi-generator-version)/openapi-generator-cli-$(openapi-generator-version).jar
+openapi-generator-jar:=target/openapi-generator-cli.jar
+openapi-generator-cli:=java -jar $(openapi-generator-jar)
+
+generator:=java
+library:=okhttp-gson
+services:=payments management balanceplatform transfers
```
Comment (michaelpaul) [inline Makefile:8]: These are services we can generate models already. New API's like Management or Balance Platform will be developed soon. Classic Payments have been upgraded in #808. The others below (e.g. recurring, payout, platforms, etc.) will be migrated in separate PR's.

File: .github/workflows/models.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Java Models
+
+on: [ workflow_dispatch, pull_request ]
```
Comment (michaelpaul) [inline .github/workflows/models.yml:3]: This event is only used for testing: 

```suggestion
on: [ workflow_dispatch ]
```

File: .github/workflows/models.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          cd target/spec
+          echo ::set-output name=pr_title::"Update models"
+          echo ::set-output name=pr_body::"OpenAPI spec or templates produced new models on $(date +%d-%m-%Y) \
+            by [commit](https://github.com/Adyen/adyen-openapi/commit/$(git rev-parse HEAD))."
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+#          token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
+#          committer: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
+#          author: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
```
Comment (michaelpaul) [inline .github/workflows/models.yml:24]: Use @AdyenAutomationBot: 

```suggestion
          token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
          committer: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
          author: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
```

**Review by AlexandrosMor — DISMISSED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]
